packages/block-library/src/site-logo/edit.js

```diff
@@ -517,11 +523,12 @@ export default function LogoEdit( {
 
 	const mediaReplaceFlowProps = {
 		mediaURL: logoUrl,
+		name: ! logoUrl ? __( 'Choose Site Logo' ) : __( 'Replace' ),
```

Comment:
Should this be title cased?

Reply:
I was drawing from the "Site Icon" terminology in WP admin, but this could just be "Choose logo". 

![CleanShot 2024-07-12 at 16 16 39](https://github.com/user-attachments/assets/6a3d398f-164d-4890-a31a-d295492688a7)

Reply:
Should we extract and discuss this more broadly? I also prefer sentence case, but committing to that will mean diverging from wp-admin. Sometimes this creates extra effort as we found in https://github.com/WordPress/gutenberg/issues/58198.

It would be good to be consistent across all areas.

Reply:
Updated it to "Choose logo"